SI also uses OriginatingEndpoint and ProcessingEndpoint headers (#5630)
Particular/docs.particular.net: nservicebus/messaging/headers.md

The endpoint name the message was sent from.

> INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebusconversationid)

Author: ```suggestion
INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#messaging-interaction-headers-nservicebus-conversationid)
```

> INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebus.conversationid)

Reviewer: ```suggestion
INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebus-conversationid)
```

> INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebus-conversationid)

Author: ```suggestion
INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebusconversationid)
```

Name of the endpoint where the message was processed.

> INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebusconversationid)

Author: ```suggestion
INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#messaging-interaction-headers-nservicebus-conversationid)
```

> INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebus.conversationid)

Reviewer: ```suggestion
INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebus-conversationid)
```

> INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebus-conversationid)

Author: ```suggestion
INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebusconversationid)
```
